contrib/kind-common

```diff
@@ -581,3 +581,24 @@ kubectl_wait_dnsnameresolver_pods() {
   echo "Waiting for pods in dnsnameresolver-operator namespace to become ready (timeout ${timeout})..."
   kubectl wait -n dnsnameresolver-operator --for=condition=ready pods --all --timeout=${timeout}s
 }
+
+deploy_primary_udn_nad() {
```

Comment:
what do you suggest ? passt_binding ?

Reply:
I don't know. I don't know what network is this. Is there a diagram somewhere?

Reply:
This is how we install the plugin-based kubevirt binding that will be selected by users that want to connect to primary UDNs from KubeVirt. 

No docs right now.

Reply:
past_binding looks fine to me.

Reply:
went for kubevirt binding, since we'll end up replacing this binding for another - i.e. that's just an implementation detail.

What matters (IMO) is we indicate what this is for - the kubevirt integration.